commands/tiering.py

```diff
@@ -525,55 +526,61 @@ async def sroom(self,
                                   required=False)):
         currentname = ctx.channel.name
         namesuffix = ""
```

Comment:
```suggestion
        namesuffix = ""
        await ctx.interaction.response.defer()
```

We need to defer the response in case the bot takes longer to respond for whatever reason. Not doing this causes a "this interaction failed to respond" error.